analytics/terraform/datahub-on-eks/README.md

```diff
@@ -0,0 +1,78 @@
+# Deploying DataHub on EKS
+Checkout the [documentation website](https://awslabs.github.io/data-on-eks/docs/) to deploy this pattern and run sample tests.
```

Comment:
change this link to https://awslabs.github.io/data-on-eks/docs/data-analytics/datahub-on-eks